File: Data Structures & Algorithms/remove-element/submission-0.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int k = nums.size();
        for(int i = 0; i < k; i++){
            if(nums[i] == val){
                nums.erase(nums.begin() + i);
                nums.push_back(val);
                i--;
                k--;
            }
        }
        return k;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}

        for i in nums:
            freq[i] = freq.get(i, 0) + 1
        
        freq_sorted = sorted(freq.items(), key = lambda item: item[1], reverse=True)
        
        res = []

        for i in range(k):
            res.append(freq_sorted[i][0])

        return res